README.md

```diff
@@ -544,34 +555,44 @@ The general steps to update the bindings to a newer version of the NVML API are
 ### Update `nvml.h`
 
 Pull down the `nvml.h` containing the updated API and commit it back to `gen/nvml/nvml.h`. The `Makefile` contains a command:
+
 ```bash
-make CUDA_VERSION=11.1 update-nvml-h
+make update-nvml-h
```

Comment:
How does it know which one to pull without this?

Reply:
Updated.